### ISSUE TITLE
ODM: digit like a property of the document

### ISSUE THREAD

**Issue by aleksandrzen** [phalcon/cphalcon#11382]
I have a collection in mongo, one of the documents in this collection has property `'18'` with value `3.56`.
I tried to select this document, but got an error: 

```
Exception Parameter 'attribute' must be a string in https://github.com/viboomru/viboom.com/blob/stable/library/Viboom/Phalcon/Mvc/Collection.php#L24

Phalcon\Mvc\Collection->writeAttribute(18, 3.56)
```

Stack trace:
```
#0 [internal function]: Phalcon\Mvc\Collection\Document->writeAttribute(18, 3.56)
#1 [internal function]: Phalcon\Mvc\Collection::cloneResult(Object(Phalcon\Mvc\Collection\Document), Array)
#2 [internal function]: Phalcon\Mvc\Collection::_getResultset(Array, Object(Models\Smth), Object(MongoDB), false)
```

**Comment by sergeyklay** [comment]
It is Phalcon `2.0.x`?

**Comment by aleksandrzen** [comment]
@sergeyklay, I've tested this in `2.0.8`

**Comment by sergeyklay** [comment]
Fixed in `2.1.x` branch

**Comment by aleksandrzen** [comment]
@sergeyklay, thx! :+1:
